packer/scripts/firstuser.sh

```diff
@@ -86,6 +95,8 @@ checkFirstUser() {
    echo "${status}"
 }
 
+stopServices
+cleanServicesPIDs
```

Comment:
Thanks for the workaround, can you also add the reason of this workaround embedded in the code as comment?
so that later , when people revisit this part of code can understand the reason.
then :+1 for me